doc/rtd/reference/custom_modules/custom_formats.rst

```diff
@@ -0,0 +1,51 @@
+.. _custom_formats:
+
+Custom Formats
+**************
+
+One can define custom data formats by presenting a `#part-handler`
+:ref:`user-data format<user_data_formats>`
```

Comment:
The link doesn't hold a lot of value to jump back to user_data_formats because it just points to our formats page which now redacts all the part-handler specific content.

Reply:
Note that there will be collisions with James' PR here too https://github.com/canonical/cloud-init/pull/5551/files

Reply:
https://github.com/canonical/cloud-init/pull/5548#issuecomment-2252327342